### PR TITLE
Update Rubocop

### PR DESCRIPTION
**Pull request by rubene**
🐘 
* Update rubocop as per security alert from gemnasium.
`RuboCop uses /tmp to store cache files insecurely. Malicious local
users could exploit this to tamper with cache files belonging to other
users.`
* Update syntax violations
screen shot 2017-05-29 at 10 03 07 am

### CODE REVIEW

**Comment by darkodosenovic** [comment]
:book:

File: spec/dummy/log/test.log [hunk 1]
```diff
@@ -0,0 +1,54 @@
+WARNING: Note, that $header-sizes has been replaced with $header-styles. $header-sizes still works, but it is going to be depreciated.
```
Comment (MathieuGilbert) [inline spec/dummy/log/test.log]: Does this file need to be checked in? Can it be gitignored?

**Comment by MathieuGilbert** [comment]
💩

File: spec/dummy/log/test.log (hunk 1 above)
Comment (rubene) [inline spec/dummy/log/test.log]: this is on the gitignore spec/dummy/log/*.log will double check

**Comment by rubene** [comment]
:wave:

**Comment by MathieuGilbert** [comment]
:elephant: :elephant: :elephant:

**Pull request merged**